### PR TITLE
Add field "text" to the object message in result (SARIF)

### PR DESCRIPTION
**Pull request by h3xstream**
Change required for Github security events support. See  #1630

---- 
**TODO**

- [x] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

File: spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Message.java [hunk 1]
```diff
@@ -13,17 +13,23 @@
...
  */
 final class Message {
+    @NonNull
+    final String text;
+
     @NonNull
     final List<String> arguments;
 
-    Message(@NonNull List<String> arguments) {
+    Message(@NonNull String text, @NonNull List<String> arguments) {
```
Comment (ThrawnCA) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Message.java:22]: This will affect any other code that might use this class.

Comment (h3xstream) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Message.java]: I removed the parameter from the constructor. Accessing the field is the only way to set the value.

Some null protection was added and the code calling it was refactored.

File: spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Message.java [hunk 2]
```diff
@@ -13,22 +13,22 @@
...
+    Message(@NonNull List<String> arguments) {
         this.arguments = Collections.unmodifiableList(Objects.requireNonNull(arguments));
     }
 
     JsonObject toJsonObject() {
         JsonObject jsonObject = new JsonObject();
         jsonObject.addProperty("id", "default");
 
-        jsonObject.addProperty("text", text);
+        if(text != null) {
```
Comment (ThrawnCA) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Message.java:29]: Should be a space between `if` and bracket.

**Review by ThrawnCA — DISMISSED** [review]

**Review by KengoTODA — CHANGES_REQUESTED** [review]
Please check the feedback from ThrawnCA, I think we can fix it automatic by `./gradlew spotlessApply`.

Then merge the latest `origin/master` to your topic branch, and resolve a conflict in the `CHANGELOG.md` file.

**Comment by h3xstream** [comment]
Everything should be there.
Sorry for the delay

**Review by ThrawnCA — DISMISSED** [review]

**Comment by h3xstream** [comment]
@KengoTODA  I've merge the CHANGELOG file.

Let me know if something more is needed.

**Review by ThrawnCA — APPROVED** [review]

**Comment by h3xstream** [comment]
@KengoTODA Let me know if anything is missing.

**Review by KengoTODA — APPROVED** [review]

**Pull request merged**